src/main/groovy/org/savantbuild/plugin/database/DatabasePlugin.groovy

```diff
@@ -163,7 +163,7 @@ class DatabasePlugin extends BaseGroovyPlugin {
       if (settings.grantUsername) {
         output.infoln("Granting privileges to [${settings.grantUsername}]")
         execAndWait(["psql", "-U", createUsername, settings.createArguments, "-c", "GRANT ALL PRIVILEGES ON DATABASE ${settings.name} TO ${settings.grantUsername}"])
-        execAndWait(["psql", "-U", createUsername, settings.createArguments, "-c", "GRANT ALL PRIVILEGES ON DATABASE ${settings.name} TO ${settings.grantUsername}"])
+        execAndWait(["psql", "-U", createUsername, settings.createArguments, "-c", "ALTER DATABASE ${settings.name} OWNER TO ${settings.grantUsername}"])
```

Comment:
presumably we can use this syntax back to PostgreSQL version 10 (I think this is what we advertise as our min. supported version)

Reply:
Verified on PG 10.21. While it's not necessary, it doesn't break anything.